### DISCUSSION TITLE
Bluesky domain handle question

### DISCUSSION THREAD

**Discussion by stoppableforce** [snarfed/bridgy#1692] [Q&A]
I'm in a weird situation that I can't figure out how to handle: my handle on Bluesky is a domain, but it is NOT the domain that I want to publish from. I found a couple of issues that looked relevant, and it seemed like maybe just adding the publishing domain to my bio before signing would be sufficient (technically bridgy suggests using the website field which.... Bluesky does not have), but so far, no dice. Is there anything I can do besides changing my handle on bsky?

Bluesky Profile: https://bsky.app/profile/awaymessage.club
Publishing from: https://notes.awaymessage.club

**Comment by snarfed** [comment]
Hmm! Very true, this is indeed awkward right now. In the short term, you can add your notes subdomain as another web site at https://brid.gy/edit-websites?source_key=agdicmlkLWd5ci0LEgdCbHVlc2t5IiBkaWQ6cGxjOnlrMjM0Z2piN3VtdXlrYTR0dWdvb243egw

If you had https://notes.awaymessage.club in your profile bio at some point when you logged into Bridgy, I'm surprised that didn't work. I'll look at that!
